test/Bootstrapper.FunctionalTests/BootstrapperTests.cs

```diff
@@ -118,7 +117,7 @@ public void BootstrapperInvokesApplicationHostWithExplicitAppBase(string flavor,
         {
             var runtimeHomeDir = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
 
-            using (var tempSamplesDir = TestUtils.PrepareTemporarySamplesFolder(runtimeHomeDir))
+            using (var tempSamplesDir = TestUtils.PrepareTemporarySamplesFolder(runtimeHomeDir, globalJson: false))
```

Comment:
Why? How do you determine solution root if you don't have a global.json? I believe after you restore for `HelloWorld`, its dependency project `HelloShared` should be present in HelloWorld/project.lock.json. How does `dnu restore` find this dependency when you don't have a solution root?